Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -682,6 +696,41 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
       public void Dispose() { }
     }
 
+    /// <summary>
+    /// Scans all the children in order of the argument Transform, appending each transform
+    /// it finds to toFill. Children are added depth-first by default.
+    ///
+    /// Pass breadthFirst: true to fill the list breadth-first instead.
+    /// </summary>
+    public static void GetAllChildren(this Transform t, List<Transform> toFill,
```

Comment:
How about GetDescendants?

Reply:
That's not very intellisense-discoverable, don't you think? Typing "GetChil-" or "children" should show this in the list of intellisense options

Reply:
GetChildrenRecursive could be more clear